.eslintrc.js

```diff
@@ -94,15 +93,21 @@ module.exports = {
         ],
       },
     ],
+    'material-ui/no-hardcoded-labels': 'off', // We are not really translating the docs/website anymore
   },
   overrides: [
+    ...baseline.overrides,
+    {
+      files: ['docs/src/modules/components/**/*.js'],
+      rules: {
+        'material-ui/no-hardcoded-labels': 'off',
+      },
+    },
     {
       files: ['examples/**/*'],
       rules: {
         // We use it for demonstration purposes
         'no-console': 'off',
-        // Personal preference
-        'no-underscore-dangle': 'off',
```

Comment:
Seems marginally used

Reply:
There is a reason. examples don't contain eslint. So the code shouldn't contain eslint comments.

Reply:
This makes sense. Should we move this rule to the share eslint config then. It feels like we want this for all /examples folders through MUI codebase.

Reply:
👍 I'm good with that as well